### PR TITLE
support multiple identical args in SCYLLA_EXT_OPTS

### PR DESCRIPTION
**Pull request by fruch**
since we have cases multiple `--experimental-features` needs to be passed via `SCYLLA_EXT_OPTS`, adding support for key with multiple values in `process_opts()`

fixes: #540
closes: #541

## Testing - to be on the safe side:
- [x] https://jenkins.scylladb.com/job/scylla-staging/job/dtest-pytest-gating/340/ (OSS)
- [x] https://jenkins.scylladb.com/job/scylla-staging/job/dtest-pytest-gating/341/ (Enterprise)

### CODE REVIEW

**Review by nyh — APPROVED** [review]
Looks good to me. I assumed you tested it actually works? :-)

File: ccmlib/scylla_node.py [hunk 1]
```diff
@@ -609,7 +609,10 @@ def process_opts(opts):
             elif k != '--memory':
                 args.append(k)
                 if v:
-                    args.append(v)
+                    args.append(v[0])
+                    for val in v[1:]:
+                        args.append(k)
+                        args.append(val)
```
Comment (tchaikov) [inline ccmlib/scylla_node.py:615]: if we really want to unify the argument to be multiple values, i'd suggest:
```py
elif k != '--memory':
    for val in v:
        args += [k, val]
```

**Comment by tchaikov** [comment]
might want to reference #541.

copy @eliransin , hi Eliran, in case you are still interested in this issue.

File: ccmlib/scylla_node.py (hunk 1 above)
Comment (fruch) [reply ccmlib/scylla_node.py:615]: yeah that a bit clearer

**Comment by fruch** [comment]
> Looks good to me. I assumed you tested it actually works? :-)

it works for the cases reported, i.e. multiple `--experimental-features`

File: ccmlib/scylla_node.py (hunk 1 above)
Comment (fruch) [reply ccmlib/scylla_node.py:615]: used that, and it's much better, thanks.

**Review by tchaikov — APPROVED** [review]
lgtm

**Review by Bouncheck — APPROVED** [review]
Looks to be working for me. My test had issues, but the cluster launches with system.tablets present and I see both experimental features in system.log

**Comment by fruch** [comment]
tests showed, that a few places were needed to change and expect a list, not assume an int.

fixed those and re-running gating

**Comment by fruch** [comment]
Code is failing in gating

Need to check why it's still breaking Scylla boot

**Comment by bhalevy** [comment]
@nyh by the way, why don't we support comma-separated list arg in scylla --experimental-features?

**Comment by nyh** [comment]
I don't know (I don't think I wrote that code?), we definitely could. But we need to also keep supporting the current approach (multiple instances of the option), because it's convenient in scripts.

**Comment by fruch** [comment]
> Code is failing in gating
> 
> Need to check why it's still breaking Scylla boot

found the issue, the args without value, was passing empty space as value (it wasn't clear from logs)
now fixe, and giving one more go in gating

**Pull request merged**